server/src/main/java/org/opensearch/search/aggregations/bucket/terms/GlobalOrdinalsStringTermsAggregator.java

```diff
@@ -603,6 +603,15 @@ abstract class ResultStrategy<
         TB extends InternalMultiBucketAggregation.InternalBucket> implements Releasable {
 
         private InternalAggregation[] buildAggregations(long[] owningBucketOrds) throws IOException {
+            int requiredSizeLocal;
```

Comment:
Super nit, I think you could start with assigment:

```suggestion
int requiredSizeLocal = bucketCountThresholds.getShardSize();
int minDocCountLocal = bucketCountThresholds.getShardMinDocCount();
```

Reply:
`minDocCountLocal` is used in the inner class below so it needs to be final. I could assign `requiredSizeLocal` and `minDocCountLocal` using ternary operators but I don't think there's much of a difference here.